modules/network-connectivity-center/variables.tf

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  description = "Project id of the project that holds the network."
+  type        = string
+}
+
+variable "ncc_hub_name" {
+  description = "The Name of the NCC Hub"
+  type        = string
+  default     = "ncc-hub"
+}
+
+variable "export_psc" {
+  description = "Whether Private Service Connect transitivity is enabled for the hub"
+  type        = bool
+  default     = true
+}
+
+variable "vpc_spokes" {
+  description = "VPC network that is associated with the spoke"
+  type = map(object({
```

Comment:
Add labels for spoke. Will be great to have an option of common labels for all spokes and then spoke specific label. You can define a variable common spoke variable and also add a line in each spoke object. Combine both of them before assigning label to every spoke.